PrivateCollections/Germany/CHJN007.xml

```diff
@@ -0,0 +1,1175 @@
+                           <textLang mainLang="gez"></textLang>
+                           <note>Numbered 140.</note>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.140">
+                           <locus from="125v"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps140"></title>
+                           <textLang mainLang="gez"></textLang>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.141">
+                           <locus target="#126v"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps141"></title>
+                           <textLang mainLang="gez"></textLang>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.142">
+                           <locus from="126v"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps142"></title>
+                           <textLang mainLang="gez"></textLang>
+                           <note>Numbered 140.</note>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.143">
+                           <locus from="127v"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps143"></title>
+                           <textLang mainLang="gez"></textLang>
+                           <note>Numbered 150.</note>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.144">
+                           <locus from="128r"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps144"></title>
+                           <textLang mainLang="gez"></textLang>
+                           <note>Numbered 150.</note>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.145">
+                           <locus from="129r"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps145"></title>
+                           <textLang mainLang="gez"></textLang>
+                           <note>Numbered 150.</note>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.146">
+                           <locus from="130r"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps146"></title>
+                           <textLang mainLang="gez"></textLang>
+                           <note>Numbered 150.</note>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.147">
+                           <locus from="130v"></locus>
+                           <title type="complete" ref="LIT2000Mazmur#Ps147"></title>
+                           <textLang mainLang="gez"></textLang>
+                           <note>Numbered 150.</note>
+                         </msItem>
+                         <msItem xml:id="p2_i1.1.148">
+                           <locus from="131r"></locus>
```

Comment:
Not now for this manuscript (I know it will take an eternity), but I think that as a general rule it would be good to avoid providing only the starting point of texts, but always to indicate the beginning and end.